src/modules/sub-items/sub.items.module.js

```diff
@@ -461,14 +461,14 @@ export default class SubItemsModule extends Component {
                 );
             } else if (hadUserContentItemFailed) {
                 modalTableTitle = Translator.trans(
-                    /*@Desc("User(s) cannot be deleted (%itemsCount%)")*/ 'bulk_delete.error.modal.table_title.users',
+                    /*@Desc("%itemsCount% User(s) cannot be deleted")*/ 'bulk_delete.error.modal.table_title.users',
                     {
                         itemsCount: notDeletedItems.length,
                     },
                     'sub_items'
                 );
                 message = Translator.trans(
-                    /*@Desc("%notDeletedCount% of the %totalCount% selected item(s) could not be deleted because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.")*/ 'bulk_delete.error.message.users',
+                    /*@Desc("%notDeletedCount% of the %totalCount% selected item(s) could not be deleted because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator.")*/ 'bulk_delete.error.message.users',
```

Comment:
```suggestion
                    /*@Desc("%notDeletedCount% of the %totalCount% selected item(s) could not be deleted because you do not have sufficient permissions. {{ moreInformationLink }} Contact your Administrator.")*/ 'bulk_delete.error.message.users',
```